### ISSUE TITLE
Force Feedback just work in one direction

### ISSUE THREAD

**Issue by Paul181516** [csutorasa/XOutput#168]
Hi i have a Nitro Tri-Force Wheel which has the sames drivers than the logitech driving force. In any game without XOutput the Force Feedback works correctly, but in FH3 with the emulator i just doesnt work. When i press "Test Forcefeedback" the wheel goes all the way right, its like the wheel can only goes to the right, not center. I think (Maybe) the problem can solve if i can invert the force feedback.
Log:
[https://drive.google.com/file/d/1teY9FJTJtBn9w9J7qz5gxRPN7zs_S0lC/view?usp=sharing](url)

[https://drive.google.com/file/d/14EcrlczIZyPDH0r8PSLJXaUm9JJtFJOq/view?usp=sharing](url)

**Comment by Paul181516** [comment]
No help? :(

**Comment by csutorasa** [comment]
There is no way to invert force. I am not sure how this can be fixed.

**Comment by Paul181516** [comment]
Yeah but the problem is that when i press "Test Force feedback" the wheel justo go all the way to the right and locks there, what can i do?

**Comment by csutorasa** [comment]
Force feedback should not affect how the axis works. Maybe it is the same issue that some experience, that controllers or settings are being dropped while the application is running.

**Comment by Hyratel** [comment]
FFB wheels and sticks have no- or a weak centering spring, instead it's a closed-loop feedback servo logical-circuit that brings the stick to center, so the FFB signal has to be Signed. the Right Lock behavior is likely due to XOutput only getting the Rumble function, which is Unsigned or positive-only amplitude and pattern because there's no benefit to running a rumble motor backwards
